examples/get_events/main.go

```diff
@@ -45,7 +45,14 @@ func demo(deployedContract *flow.Account, runScriptTx *flow.Transaction) {
 	printEvents(result, err)
 
 	// Query for our custom event by type
-	customType := fmt.Sprintf("AC.%s.EventDemo.EventDemo.Add", deployedContract.Address.Hex())
+	//customType := fmt.Sprintf("AC.%s.EventDemo.EventDemo.Add", deployedContract.Address.Hex())
```

Comment:
Remove this please

Reply:
removed here https://github.com/onflow/flow-go-sdk/pull/449/commits/155a11e76499d84ef5487b8d1ca7cd29d57cf428